### ISSUE TITLE
Redux Builder

### ISSUE THREAD

**Issue by gulonline** [reduxframework/redux-framework-4#168]
https://build.redux.io/ is not working, getting 404 error.

**Comment by dovy** [comment]
Yes, I fear the builder has been down for some time now. It will be rebuilt sometime in the future, but only for Redux Pro Elite subscribers.  ;)

**Comment by gulonline** [comment]
I think this is a false idea. Because if you keep it paid, then how will you explore your projects? After this, if someone likes your system (https://build.redux.io), then it will go to the paid version. If you keep it paid from scratch, then how will anyone use it.

I myself am a user of your system (https://build.redux.io) since last 5+ years. And ever since I have used your system, I have not used any other system. But now when you are saying that this system (https://build.redux.io) will be paid, then I feel that I should switch to another system now. Now you think that your 5+ year old member is getting out of your system.

So my request to you is that before making paid version  (https://build.redux.io), you must think about users like us once.

Thank you.

**Comment by dovy** [comment]
@gulonline Let me put this into perspective for you, and please realize I mean no disrespect.

I originally started working on a full drag and drop panel building. Allowing people to build out panels without having to code the framework at all. It worked quite well, but cost me quite a lot of money hiring a developer to work with me.

Now that GitHub has changed their API and PHP has made such drastic improvements, my old code won't work anymore for the builder. Which means I will have countless hours to rebuild if I do it myself, and countless hours of contractor work if I hire someone else to work with me.

If it was easy as turning the builder back on I would. But the truth of the matter is, it's going to cost me money to make it work again. Either in my personal hours (when I make good money) or a contractor (which also makes more money). I cannot take that on without $$$ either way.

Please understand, this isn't a bait and switch. It will cost lots of money and time. As such, I can't just give it away when I get around to developing it. I have to justify the effort somehow. I hope you understand.

**Comment by gulonline** [comment]
dear @dovy I agree with you 100%. You are right but as you said that you have worked hard for years in this way, we have also worked hard to understand your system and use it in WordPress, and now with your paid version, we have time to understand another system and again it will take.

As in the last few years I have realized that after understanding your system, there will be no need for any other system. But I got a bit disappointed with your decision to go from Open Source to paid version. So there is a request that **_if possible_**, just keep the option of generating blank template free so that a beginner developer like me can be easy to understand.

thank you for your great work.

thanks & regards.
